### PR TITLE
Add Tini to dockerfile

### PR DESCRIPTION
**Pull request by KeithTt**
https://github.com/siddontang/go-mysql-elasticsearch/issues/233

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -6,7 +6,7 @@ COPY . /go/src/github.com/siddontang/go-mysql-elasticsearch
 
 RUN cd /go/src/github.com/siddontang/go-mysql-elasticsearch/ && \
     go build -o bin/go-mysql-elasticsearch ./cmd/go-mysql-elasticsearch && \
-    cp -f ./bin/go-mysql-elasticsearch /go/bin/go-mysql-elasticsearch
-
-ENTRYPOINT ["go-mysql-elasticsearch"]
+    cp -f ./bin/go-mysql-elasticsearch /go/bin/go-mysql-elasticsearch && \
+    apk add --no-cache tini mariadb-client
```
Comment (siddontang) [inline Dockerfile]: I prefer using a separated command for the library.

File: Dockerfile [hunk 2]
```diff
@@ -6,7 +6,7 @@ COPY . /go/src/github.com/siddontang/go-mysql-elasticsearch
...
 
+ENTRYPOINT ["/sbin/tini","--","go-mysql-elasticsearch","-config=/go/src/github.com/siddontang/go-mysql-elasticsearch/etc/river.toml"]
```
Comment (siddontang) [inline Dockerfile]: How can the users use the customized config? maybe we should remove `-config` here.

Another way is use volumn to let the user bind the config.

File: Dockerfile (hunk 1 above)
Comment (KeithTt) [reply Dockerfile]: You mean put ʻapk addʻ in an separate RUN command?

File: Dockerfile (hunk 2 above)
Comment (KeithTt) [reply Dockerfile]: Maybe we could add a COPY or ADD command to copy the customized config to the container, like this:

```
COPY ./river.toml /go/src/github.com/siddontang/go-mysql-elasticsearch/etc/river.toml
```

Comment (KeithTt) [reply Dockerfile]: After a short thought, I think it is better to remove `-config`..

File: Dockerfile [hunk 3]
```diff
@@ -2,11 +2,12 @@ FROM golang:alpine
...
 
-COPY . /go/src/github.com/siddontang/go-mysql-elasticsearch
+ADD . /go/src/github.com/siddontang/go-mysql-elasticsearch
 
 RUN cd /go/src/github.com/siddontang/go-mysql-elasticsearch/ && \
     go build -o bin/go-mysql-elasticsearch ./cmd/go-mysql-elasticsearch && \
     cp -f ./bin/go-mysql-elasticsearch /go/bin/go-mysql-elasticsearch
 
-ENTRYPOINT ["go-mysql-elasticsearch"]
+RUN apk add --no-cache tini mariadb-client
```
Comment (siddontang) [inline Dockerfile]: can we move it up before L5? So we can still use the docker cache even we change the source.

**Comment by siddontang** [comment]
Rest LGTM

File: Dockerfile (hunk 3 above)
Comment (KeithTt) [reply Dockerfile]: OK!

**Pull request merged**